ogr/ogrsf_frmts/openfilegdb/ogropenfilegdblayer.cpp

```diff
@@ -664,7 +664,7 @@ int OGROpenFileGDBLayer::BuildLayerDefinition()
         // string width is 0, we pick up 65536 by default to mean unlimited
```

Comment:
this comment needs updates, maybe
```suggestion
        // string width is 0, if OPENFILEGDB_DEFAULT_STRING_WIDTH is not set we pick up 65536 by default to mean unlimited
```